include/sys/mntent.h

```diff
@@ -80,8 +80,8 @@
 #define	MNTOPT_SETUID	"setuid"	/* Set uid allowed */
 #define	MNTOPT_NOSETUID	"nosetuid"	/* Set uid not allowed */
 #elif defined(__APPLE__)
-#define	MNTOPT_SETUID	"setuid"	/* Set uid allowed */
-#define	MNTOPT_NOSETUID	"nosetuid"	/* Set uid not allowed */
+#define	MNTOPT_SETUID	"suid"		/* Set uid allowed */
+#define	MNTOPT_NOSETUID	"nosuid"	/* Set uid not allowed */
```

Comment:
Did this undo the #ifdef apple (you will translate later), then we can drop the #ifdef?